README.md

```diff
@@ -5,10 +5,21 @@
   <br />
 </p>
 
+[![Legacy SDK](https://img.shields.io/badge/state-legacy-yellow)](https://github.com/amplitude/Amplitude-Swift)
```

Comment:
It's kinda of confusing that the legacy badge points to the new SKD repo. But I checked the old Android, it's the same. So this is how we do it.